src/browser/window_groups.ts

```diff
@@ -72,6 +72,11 @@ export class WindowGroups extends EventEmitter {
         return hash.digest('hex');
     }
 
+    //cannot rely on nativeId as windows might leave a group after they are closed.
+    private getWindowGroupId = (identity: Identity): string => {
+        return `${identity.uuid}:${identity.name}`;
```

Comment:
have these already been encoded so that `:` cant cause a problem?

Reply:
also, dont we in other places use `/` when making UUID/name as a single string? Ought this not be standardized (including escaping or replacing the separator char)?